view/src/components/filters.js

```diff
@@ -49,8 +56,15 @@ class Filters extends Component {
     url += `locationId=${this.state.locationId}`;
     url += "&";
     url += `distance=${this.state.distance}`;
+    console.log("URL", url);
+    // axios.defaults.headers.common = { Authorization: `${this.getAuth()}` };
     axios.get(url).then((res) => {
-      this.setState({ results: res.data });
+      this.setState(
+        { results: res.data },
+        console.log("results[]:", res.data, "res", res)
+      );
+      // make this async next
+      return this.props.render(this.state.results);
```

Comment:
remove commented out code and console logs

Also that last comment makes me think there's still work to do? Remove?

Reply:
(I should've pushed this commit on PR 58. Removed these as requested)